src/shared/file_op.c

```diff
@@ -2029,8 +2029,12 @@ char **expand_win32_wildcards(const char *path) {
             if (hFind == INVALID_HANDLE_VALUE) {
                 long unsigned errcode = GetLastError();
                 if (errcode == 2) {
-                    mdebug2("No file/folder that matches %s.", pattern);
-                } else {
+                    mdebug2("No file that matches %s.", pattern);
+                }
+                else if (errcode == 3) {
```

Comment:
To comply with [code style guide](https://github.com/wazuh/wazuh/wiki/Coding-style-guide-%28C%29#style-conventions):
```suggestion
                } else if (errcode == 3) {
```